### DISCUSSION TITLE
There is no way to change shortcuts for switching between different tabs?

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#815] [Ideas]
### What feature would you like to see?

Currently Alt+number switches between tabs, and it doesn't look like there is a way to change that? I have my ctrl and alt keys swapped, so I would like to be able to change the shortcut to be ctrl+number instead. 

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/508_

**Comment by vrepinsky** [comment]
Bumping that, would be really nice to be able to use CMD + Option + Arrows to navigate quickly between tabs (naturally vertical) and workspaces (left and right). Currently in Zen we've got default Firefox navigation between tabs which is designed with horizontal alignment of tabs in mind, and the shortcut can't be modified via settings

**Comment by mfpiccolo** [comment]
I changed my workspace shorcut to command + control LeftArrow/RightArrow. Would be great if I could make command + control + UpArrow and DownAarow nav up and down the tabs

**Comment by benyamynbrkyc** [comment]
Bump! It would be great if we could have more tab switching options! Also, would it be possible to be able to customize e.g. the default Ctrl+Tab keybinding without using something like Karabiner?

**Reply by krauterbaquette** [reply]
I am quite positiv this is possible.
I rebound `New Tab` to `Alt + Shift + T` in the settings under "keyboard shortcuts".
Now when I press `Ctrl + T` nothing happens

**Reply by shadizx** [reply]
This does not update the tab cycle command though.

**Comment by mertgoksel** [comment]
Also appears that we cannot change ctrl+tab keybind to switch tabs in order. That keybind is very weird to press so i would like to change it.

**Comment by Ruslanmsv** [comment]
Bumping this one up, so we are able to change ctrl+tab to another key binding

Use case: 
1. I have my Caps Lock remaped to Ctrl in macos settings
2. I used **Ctrl + K** to cycle forward between recent tabs and **Ctrl + T** to cycle backwards on Arc browser 
3. I want to configure that for Zen/Twilight

Unfortunately, I can not remap Ctrl+Tab in Zen/Twilight. 
If i go forward with Ctrl + Tab in Zen/Twilight, i can not go backwards, unless i use uncomfortable Cmd+Shift+Tab shortcut

**Comment by shadizx** [comment]
There are shortcuts for literally everything besides this - this should definitely be a part of that last. We've been asking for almost a year now, how can we escalate this?

**Comment by nilognap** [comment]
modifier + h/l for going back/forward on pages
modifier + j/k for switching between tabs (since they are vertical)
some other modifiers + h/l for switching between workspaces

or something like that, would be PERFECTION

**Reply by mpHofa** [reply]
yes so much this.

**Comment by macbaydigital** [comment]
In Firefox on Mac, I switch between tabs using Cmd + Option + Left or Right Arrow. I tried applying the same shortcuts in Zen Browser but couldn’t find a corresponding option in the Keyboard Shortcuts settings. Any one who knows where to find and how to change the current shortcut (tab + shift)?

**Comment by mpHofa** [comment]
+1 for adding the possibility of remapping next tab and previous tab. This is essential before i can switch.
